content/lang-support/stdlib.md

```diff
@@ -136,35 +137,36 @@ sync |  <span style="color: green">✔</span> yes  |
 sync/atomic |  <span style="color: green">✔</span> yes  | 
 syscall |  <span style="color: green">✔</span> yes  | 
 syscall/js |  <span style="color: green">✔</span> yes  | 
-testing |  [<span style="color: red">✗</span> no](#testing)  | 
-testing/iotest |  <span style="color: green">✔</span> yes  | 
+testing |  <span style="color: green">✔</span> yes  | 
+testing/iotest |  [<span style="color: red">✗</span> no](#testing-iotest)  | 
```

Comment:
(Not commenting on this PR but rather on the TinyGo 0.14 release)

That looks like a regression, but probably not important.

Reply:
Agreed, or might be is correctly failing import now?